### DISCUSSION TITLE
是否可以实现推送服务？

### DISCUSSION THREAD

**Discussion by champion-yang** [nonebot/discussions#19] [问答] [answered]
nonebot 是否可以实现qq群的推送服务？望各位大佬解答~

**Comment by stdrc** [comment, marked as answer]
具体点？

**Reply by champion-yang** [reply]
具体场景说明：机器人服务接入qq群，服务器启动定时任务，定时推送消息（比如天气变化情况）给qq群。消息内容是从别的地方获取来的，也是通过定时任务获取的。
时间紧急，我没有做预研。我现在没有具体去看 nonebot 机器人服务是否可以主动推送消息给qq群，我理解应该是可以。想和各位大佬确认一下~

**Reply by stdrc** [reply]
可以的，用 scheduler
